src/learn/qml/MuseScore/Learn/internal/ClassesPage.qml

```diff
@@ -63,141 +64,159 @@ FocusScope {
         }
     }
 
-    Rectangle {
-        anchors.top: parent.top
-        anchors.left: parent.left
-        anchors.leftMargin: root.sideMargin
+    ScrollBar.vertical: StyledScrollBar {
+        parent: root.parent
+
+        anchors.top: root.top
+        anchors.bottom: root.bottom
         anchors.right: parent.right
-        anchors.rightMargin: root.sideMargin
 
-        height: authorInfo.height
+        visible: root.contentHeight > root.height
+        z: 1
+    }
 
-        radius: 12
+    header: Item {
```

Comment:
I might be missing something, but it looks like you're making it unnecessarily complicated. Why using a GridView with no items and only a header? Why not just `StyledFlickable`?

Reply:
Also, if you change it to `StyledFlickable`, I think you won't need to put everything inside this `Item` anymore.

Reply:
I was completely new to QML when I opened this file. At that time, I simply relied on trial and error to explore what worked.
I will try to fix this today.